### DISCUSSION TITLE
Project License

### DISCUSSION THREAD

**Discussion by NeverBehave** [erebe/wstunnel#293] [Q&A] [answered]
Looking at the license of the project, it seems to be a https://spdx.org/licenses/BSD-3-Clause-Clear.html. 

However it is missing the `NO EXPRESS OR IMPLIED LICENSES TO ANY PARTY'S PATENT RIGHTS ARE GRANTED BY THIS LICENSE` wording about patent. Is this intended?

**Comment by erebe** [comment, marked as answer]
Indeed it should be BSD3 new or revised, I have updated the LICENSE to make it clear.
Regarding, the question, I re-checked at  https://opensource.org/license/bsd-3-clause and the liscense does not seem to include the metionned wording.

**Reply by NeverBehave** [reply]
thanks for the clarification, I matched the clear version because of the the line `All rights reserved.`, and also it does not appear in BSD 3 clause you mentioned…? I am not sure whether this would be a big deal tho.
